cnf-tests/docgen/e2e.json

```diff
@@ -102,6 +102,22 @@
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Egress DENY all traffic from a pod": "Verifies that an Engress can deny the outgoing traffic from a specific pod",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration IPv6 ALLOW traffic to nsX_podA only from nsX_podB": "Verifies that multinetwork policy rules can be applied to IPv6 secondary networks",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic to nsX_podA only from (namespace == nsY)": "Verifies that an Ingress rules can limit incoming traffic from specific namespaces",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic to nsX_podA only from (namespace IN {nsY, nsZ} AND pod IN {podB, podC})": "Verifies that an Ingress rules can limit incoming traffic from specific namespaces and pod labels",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic to nsX_podA only from (nsY/* OR */podB)": "Verifies that an Ingress rules can limit incoming traffic from specific namespaces or pod labels",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic to nsX_podA only from nsX_podB": "Verifies that an Ingress rules can limit incoming traffic from specific pods",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress DENY all traffic to a pod": "Verifies that an Ingress rules can deny any incoming traffic",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress DENY all traffic to and within a namespace": "Verifies that an Ingress rules can deny all traffic in a namespace",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ports/Protocol Allow access only to a specific UDP port from any pod": "Verifies rules can be applied to specific UDP ports",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ports/Protocol Allow access only to a specific port/protocol SCTP [sctp]": "Verifies rules can be applied to SCTP transport protocol",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ports/Protocol Allow access only to a specific port/protocol TCP": "Verifies rules can be applied to TCP transport protocol",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ports/Protocol Allow access only to a specific port/protocol TCP+UDP": "Verifies rules can be applied to multiple transport protocols",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ports/Protocol Allow access only to a specific port/protocol UDP": "Verifies rules can be applied to UDP transport protocol",
+    "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Stacked policies enforce multiple stacked policies with overlapping selector [nsX_podA \u003c=\u003e (nsY/* OR */podB)]": "Verifies multiple rules can be applied to the same pods",
```

Comment:
What is it \u003c=\u003e?
Maybe better  remove from test name spicific pod and ns names  and leave it more general? for example
 "[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic to nsX_podA only from nsX_podB"
=>
"[sriov] [multinetworkpolicy] MultiNetworkPolicy integration Ingress ALLOW traffic from specific pod"

Reply:
Right! I removed any pod or ns specific reference from test decriptions.

Reply:
Great, thanks!